### ISSUE TITLE
Strange button behavior on iPhone for embedded viewer

### ISSUE THREAD

**Issue by pearcetm** [openseadragon/openseadragon#2641]
I'm having a very weird issue where the inactive activation states for the viewer buttons are not being appropriately hidden. It is happening specifically on iPhone on a particular embedded page. See screenshot to see what I mean:

![IMG_1075](https://github.com/user-attachments/assets/d2b01a14-0ea1-4600-95e4-ab6557e8f180)

I am befuddled by this behavior, since the visibility state of the buttons is controlled by inline CSS. Embedding the viewer into this page works fine on desktop, and when I simulate iPhone in dev tools, it still works fine. I have asked some colleagues if they are having problems, and have not gotten anyone else who is seeing this. Unfortunately I do not have their device details.

Has anyone else seen this kind of buggy button behavior before? Any advice on where to look to try to fix it?

**Comment by pearcetm** [comment]
I was able to hook my phone up to my computer to take a look at the HTML. Interestingly, the inline `style` property on the images is entirely missing:

<img width="312" alt="image" src="https://github.com/user-attachments/assets/e3c93a5a-eecd-4f2f-8b40-f5a7ea97608c" />

This definitely explains why the page looks like it does, but I still don't know why the style property is missing.

**Comment by pearcetm** [comment]
I was able to narrow this down. It turns out that the `mirage2` script from Cloudflare (imported from elsewhere in this particular content managment system, I have no control over it) was messing with the images, including stripping the inline styles.

I was able to work around this by creating my viewer within a `window.setTimeout()` call. A delay of 15 ms before creating the viewer is sufficient to fix the problem; a delay of 0 ms (just deferring to the next loop) was not.
